libs/product/driver/magento/src/interfaces/config.interface.ts

```diff
@@ -0,0 +1,18 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * The token used to provide @daffodil/product/driver/magento config data.
+ * Mandatory for the Magento driver.
+ */
+export const MAGENTO_PRODUCT_CONFIG_TOKEN = new InjectionToken<MagentoProductConfig>('MAGENTO_PRODUCT_CONFIG_TOKEN');
+
+/**
+ * An interface for providing @daffodil/product/driver/magento with necessary config values.
+ */
+export interface MagentoProductConfig {
```

Comment:
This should probably be `DaffProductMagentoDriverConfig` for consistency.

Reply:
changed